lib/tasks/check_content_consistency.rake

```diff
@@ -0,0 +1,29 @@
+namespace :check_content_consistency do
+  def check_content(content_id, locale)
+    checker = ContentConsistencyChecker.new(content_id, locale)
+    errors = checker.call
+
+    if errors.any?
+      puts "#{content_id} 😱"
+      puts errors
+    end
+
+    errors.none?
+  end
+
+  desc "Check documents for consistency with the router-api and content-store"
+  task :one, [:content_id, :locale] => [:environment] do |_, args|
+    content_id = args[:content_id]
+    locale = args[:locale] || "en"
+    check_content(content_id, locale)
+  end
+
+  desc "Check all the documents for consistency with the router-api and content-store"
+  task all: :environment do
+    documents = Document.pluck(:content_id, :locale)
+    _, failures = documents.partition do |content_id, locale|
```

Comment:
might be a bit clearer with `failures=documents.reject ...` rather than `_, failures = documents.partition ...` (sorry, i made the stupid suggestion in the first place!)